llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -121,23 +121,45 @@ static void writeToFile(std::string Filename, std::string Content) {
   OS.close();
 }
 
+// Describes scope covered by each entry in the module-kernel map populated by
+// the function below.
```

Comment:
Hmm, what if someone would come and made this comment invalid by adding new function after this enum...
```suggestion
// Describes scope covered by each entry in the module-kernel map populated by
// the collectKernelModuleMap function.
```

Reply:
ok

Reply:
ok